app/models/package_manager/packagist.rb

```diff
@@ -83,6 +83,7 @@ def self.mapping(raw_project)
         keywords_array: Array.wrap(latest_version["keywords"]),
         licenses: latest_version["license"]&.join(","),
         repository_url: repo_fallback(latest_version["source"]&.fetch("url"), latest_version["homepage"]),
+        versions: raw_project["versions"],
```

Comment:
since I was in here recently, I think you'd have to do this to get the versions now bc `raw_project` will just be a list of sorted versions:

```suggestion
        versions: raw_project.pluck("version"),
```

(example payload [here](﻿﻿﻿https://repo.packagist.org/p2/tiat/router.json))